src/redux/services/helm.ts

```diff
@@ -40,27 +46,53 @@ export function getHelmChartFromFileEntry(fileEntry: FileEntry, helmChartMap: He
  * Checks if the specified path is a helm values file
  */
 
-export function isHelmValuesFile(filePath: string) {
-  return micromatch.isMatch(path.basename(filePath), '*values*.yaml');
+export function isHelmValuesFile(filePath: string): boolean {
+  return micromatch.isMatch(path.basename(filePath).toLowerCase(), '*values*.yaml');
+}
+
+/**
+ * Checks if the specified path is a helm chart file
+ */
+
+export function isHelmChartFile(filePath: string): boolean {
+  return path.basename(filePath).toLowerCase() === 'chart.yaml';
 }
 
 /**
  * Checks if the specified files are a Helm Chart folder
  */
 
-export function isHelmChartFolder(files: string[]) {
+export function isHelmChartFolder(files: string[]): boolean {
   return files.indexOf(HELM_CHART_ENTRY_FILE) !== -1;
 }
 
 /**
- * check if the k8sResource is supported
- * @param resource
- * @returns @boolean
+ * check if the k8sResource is supported - currently excludes any files
+ * that seem to contain Helm template or Monokle Vanilla template content
  */
+
 export function isSupportedHelmResource(resource: K8sResource): boolean {
   const helmVariableRegex = /{{.*}}/g;
+  const vanillaTemplateVariableRegex = /\[\[.*]]/g;
+  return !resource.text.match(helmVariableRegex)?.length && !resource.text.match(vanillaTemplateVariableRegex)?.length;
+}
+
+/**
+ * Adds the values file at the given path to the specified HelmChart
+ */
 
-  return Boolean(resource.text.match(helmVariableRegex)?.length) === false;
+export function createHelmValuesFile(fileEntry: FileEntry, helmChart: HelmChart, helmValuesMap: HelmValuesMapType) {
+  const helmValues: HelmValuesFile = {
+    id: uuidv4(),
+    filePath: fileEntry.filePath,
+    name: fileEntry.filePath.substring(path.dirname(helmChart.filePath).length + 1),
```

Comment:
ins't `path.basename(fileEntry.filePath)` enough here?

Reply:
no - this is meant to be relative to the containing HelmChart - so the name would be "subfolder/values.yaml" for a values file in a subfolder - not sure what else we could/should use as the name as just the filename might be confusing if the same name is used in different folders